packages/binary-heap/binary-heap.js

```diff
@@ -0,0 +1,3 @@
+export { MaxHeap } from './max-heap';
```

Comment:
Can we adjust the local file import references here (and in the other files) to follow the same extension format - either all with a `.js` extension or without? Most of the Meteor codebase uses the `.js` extension, so that's probably the best choice for consistency.